### DISCUSSION TITLE
storybook 6+

### DISCUSSION THREAD

**Discussion by bzappasodi** [storybookjs/addon-designs#149] [General]
This does not currently work with storybook 6+ args, correct??

**Comment by pocka** [comment]
This addon supports parameters, not Args.

> A story is a component with a set of arguments that define how the component should render. “Args” are Storybook’s mechanism for defining those arguments in a single JavaScript object. Args can be used to dynamically change props, slots, styles, inputs, etc.

https://storybook.js.org/docs/react/writing-stories/args

> Parameters are a set of static, named metadata about a story, typically used to control the behavior of Storybook features and addons.

https://storybook.js.org/docs/react/writing-stories/parameters

See also #142

**Reply by bzappasodi** [reply]
So, storybook 6.4.20 is out of the question for integrating with Figma. Storybook 6.4.20 uses args rather than parameters. Correct?

**Reply by pocka** [reply]
No. You can use Parameters and Args at the same time.

Args and Parameters are completely separated features of Storybook. Args are dynamic parameters mainly used for controlling the story contents you write. On the other hand, Parameters are static parameters for other SB features and addons.
https://storybook.js.org/docs/react/writing-stories/parameters

**Reply by bzappasodi** [reply]
I see. And my main.js does not seem to be importing storybook-addon-designs
module.exports = {
  "stories": [
    "../src/**/*.stories.mdx",
    "../src/**/*.stories.@(js|jsx|ts|tsx)"
  ],
  "addons": [
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/addon-interactions",
    "@storybook/preset-create-react-app",
    '@storybook/addon-a11y',
    'storybook-addon-designs',
  ],
  "framework": "@storybook/react",
}

**Reply by pattimcletchie** [reply]
@bzappasodi did you figure out how to use this with storybook 6.4+?
